main.py

```diff
@@ -165,5 +165,12 @@ def createManagerSale():
     newSalePrice = request.args.get('newPrice')
     createSale(productId, newSalePrice)
     return redirect("/managerPage")
+
+
+@app.route("/cartClear")
+def clearCart():
+    session["resultList"] = []
+    session["totalAmount"] = 0.00
```

Comment:
Does this work? In the user login package I have it being set to `0`. I'm not sure if it makes a difference though.